torch/_inductor/codegen/common.py

```diff
@@ -306,6 +306,47 @@ def boolean_ops():
 }
 
 
+def _deduce_output_dtype_by_name(
```

Comment:
It is used by other python files. Better to name it public.
```suggestion
def deduce_output_dtype_by_name(
```

Reply:
Thanks for the comment and changed.